features/adressering/aanhef/overzicht.feature

```diff
@@ -0,0 +1,96 @@
+# language: nl
+
+@info-api
+Functionaliteit: leveren van de aanhef van een persoon
+  Als consumer
+  wil ik de aanhef (introducerende zin van een brief of e-matching) van een persoon kunnen vragen
+  zodat ik de persoon op de juiste en consistente manier kan aanschrijven
+
+  De volgende afkortingen worden gebruikt in deze specificatie:
+  - GA = "Geachte mevrouw", "Geachte heer" of "Geachte VL" afhankelijk van het geslacht
+  - VL = voorletters
+  - VV = voorvoegsel.
+  - GN = geslachtsnaam
+  - VP = voorvoegsel partner
+  - GP = geslachtsnaam partner
+
+  Regel: de aanhef van een persoon zonder partner wordt als volgt samengesteld: GA VV GN. Wanneer een persoon geen voorvoegsel heeft, wordt de overbodige spatie niet opgenomen
+
+    Abstract Scenario: persoon met voornamen, geslachtsnaam en geen voorvoegsel heeft geslacht <voorbeeld>
+      Gegeven de persoon 'Jo' met burgerservicenummer '000000141'
```

Comment:
waarom geven we hier eigenlijk een burgerservicenummer op? Kunnen we niet onderwater een burgerservicenummer toekennen?
1e persoon: 000000012
2e persoon: 000000024
enz.

Reply:
Gaan we doen, maar moet worden geimplementeerd in nieuwe stapdefinitie: automatisch genereren van de BSN als deze niet wordt opgegeven

Reply:
done